src/Mapbender/PrintBundle/Component/PrintService.php

```diff
@@ -890,6 +898,10 @@ private function drawMultiPolygon($geometry, $image)
 
     private function drawLineString($geometry, $image)
     {
+        $quality = $this->data['quality'];        
```

Comment:
@astroidex, `resize` variable isn't initalized if `quality == 72`. Please fix it.